blog.html

```diff
@@ -0,0 +1,63 @@
+        </article>
+        <article>
+          <img src="./images/blog-2.jpg" alt="Woman and dog sitting on mountaintop with backs to viewer">
+          <h2> How To Walk Multiple Dogs</h2>
+          <p>Lotsa pats noodle horse very good spot clouds fat boi, lotsa pats long bois. Shoob extremely cuuuuuute woofer boofers, corgo pupperino super chub, borking doggo stop it fren. Long bois thicc noodle horse such treat smol borking doggo with a long snoot for pats sub woofer doggo, shoober many pats clouds stop it fren. Fluffer puggo h*ck I am bekom fat mlem waggy wags, long snoot for pats the neighborhood pupper woofer.</p>
+          <p>Bork long woofer I am bekom fat ur givin me a spook vvv, long bois very hand that feed shibe shooberino. Shoober adorable doggo you are doin me a concern adorable doggo very good spot tungg dat tungg tho wrinkler wow very biscit doge, very jealous pupper he made many woofs doggo maximum borkdrive waggy wags clouds h*ck. Floofs big ol pupper super chub smol many pats, doing me a frighten big ol pupper.</p>
+        </article>
+        <article>
+          <img src="./images/blog-3.jpg" alt="Woman and dog sitting on mountaintop with backs to viewer">
+          <h2> Teach Your Dog To Fetch!</h2>
+          <p>Lotsa pats noodle horse very good spot clouds fat boi, lotsa pats long bois. Shoob extremely cuuuuuute woofer boofers, corgo pupperino super chub, borking doggo stop it fren. Long bois thicc noodle horse such treat smol borking doggo with a long snoot for pats sub woofer doggo, shoober many pats clouds stop it fren. Fluffer puggo h*ck I am bekom fat mlem waggy wags, long snoot for pats the neighborhood pupper woofer.</p>
+          <p>Bork long woofer I am bekom fat ur givin me a spook vvv, long bois very hand that feed shibe shooberino. Shoober adorable doggo you are doin me a concern adorable doggo very good spot tungg dat tungg tho wrinkler wow very biscit doge, very jealous pupper he made many woofs doggo maximum borkdrive waggy wags clouds h*ck. Floofs big ol pupper super chub smol many pats, doing me a frighten big ol pupper.</p>
+        </article>
+      </section>
+    </main>
+    <footer>
+      <img src="./images/adoptadog-logo.svg" alt="Adoptadog logo" width="200">
+      <p class="white">Doge many pats snoot super chub maximum borkdrive stop it fren, super chub blep sub woofer long doggo.</p>
+      <nav>
+        <ul class="no-bullets">
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="./dogs.html">Dogs</a></li>
+          <li><a href="./blog.html">Blog</a></li>
+        </ul>
+      </nav>
+      <p class="white">© 2018 Adoptadog, LLC | Seattle, WA</p>
```

Comment:
a good way to handle the copyright is to use the html symbol `&copy;` it is just a little more pro